### PR TITLE
[config] add missing RETRY_UNIX_SOCKET_AFTER

### PR DESCRIPTION
**Pull request by nymkappa**
Add missing config/fix tests for new config variable introduced in https://github.com/mempool/mempool/pull/3643

### CODE REVIEW

**Review by softsimon — APPROVED** [review]
utACK

Looks ok , tests are OK

File: backend/src/__fixtures__/mempool-config.template.json [hunk 1]
```diff
@@ -45,7 +45,8 @@
   },
   "ESPLORA": {
     "REST_API_URL": "__ESPLORA_REST_API_URL__",
-    "UNIX_SOCKET_PATH": "__ESPLORA_UNIX_SOCKET_PATH__"
+    "UNIX_SOCKET_PATH": "__ESPLORA_UNIX_SOCKET_PATH__",
+    "RETRY_UNIX_SOCKET_AFTER": "__RETRY_UNIX_SOCKET_AFTER__"
```
Comment (softsimon) [inline backend/src/__fixtures__/mempool-config.template.json:49]: must prepend with `__ESPLORA_`

File: docker/backend/mempool-config.json [hunk 2]
```diff
@@ -43,7 +43,8 @@
   },
   "ESPLORA": {
     "REST_API_URL": "__ESPLORA_REST_API_URL__",
-    "UNIX_SOCKET_PATH": "__ESPLORA_UNIX_SOCKET_PATH__"
+    "UNIX_SOCKET_PATH": "__ESPLORA_UNIX_SOCKET_PATH__",
+    "RETRY_UNIX_SOCKET_AFTER": "__RETRY_UNIX_SOCKET_AFTER__"
```
Comment (softsimon) [inline docker/backend/mempool-config.json:47]: Prepend with `__ESPLORA`

**Comment by nymkappa** [comment]
@softsimon I've addressed your comments in dfe2cf631facbbc50c87094be7fda999e2e5c641

**Pull request merged**